android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -358,16 +358,20 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 return true;
 
             case R.id.action_share:
+                String speakers = "";
+                for(int i = 0; i < session.getSpeakers().size(); i++){
+                    speakers +=  ((i == 0)?(""):(", ")) + session.getSpeakers().get(i).getName();
```

Comment:
We should use ```StringBuilder``` here for concatenation since it is the recommended way in case of a loop. 
@iamareebjamal What do you think ?

Reply:
Yes true